lib/Dialect/Torch/IR/TorchOps.cpp

```diff
@@ -730,6 +730,25 @@ OpFoldResult AtenToDtypeOp::fold(ArrayRef<Attribute> operands) {
   return getOperand(0);
 }
 
+//===----------------------------------------------------------------------===//
```

Comment:
We don't need this. There is a canonicalize pass after decompose complex ops pass. https://github.com/llvm/torch-mlir/blob/25ab8fcc1f5307258b7f3dbc2a91424e56a0f735/lib/Dialect/Torch/Transforms/Passes.cpp#L136 The canonicalize pass will trigger the canonicalization of `aten.to.dtype` which is already covered.